hbs-helpers.js

```diff
@@ -1,23 +1,51 @@
 "use strict";
 
+const { LocaleUtils } = require("./locale-utils");
 const mozlog = require("./log");
 
 
 const log = mozlog("hbs-helpers");
 
-function breachDataClasses(dataClasses) {
+
+function fluentFormat (req, id, args) {
+  return LocaleUtils.fluentFormat(req.supportedLocales, id, args);
+}
+
+function getValue (req, key, number, args) {
+  const fluentID = `${key}${number}`;
+  return LocaleUtils.fluentFormat(req.supportedLocales, fluentID, args);
+}
+
+function breachResponse(req, id, breachTitle, breachCount) {
```

Comment:
@stasm @flodolo Trying unsuccessfully to pass breachTitle and breachCount to the app.ftl. Any thoughts on what is going wrong here?

Reply:
This is what comes back: <img width="1002" alt="screen shot 2018-10-11 at 3 56 43 pm" src="https://user-images.githubusercontent.com/22355127/46836907-f67da880-cd78-11e8-8a14-f6fb77e99d4b.png">